<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AttachmentsList</class>
 <widget class="QWidget" name="AttachmentsList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QListView" name="mListView">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
    </widget>
   </item>
  </layout>
  <action name="mRenameAction">
   <property name="text">
    <string>Rename</string>
   </property>
   <property name="toolTip">
    <string>Rename</string>
   </property>
  </action>
  <action name="mDeleteAction">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="mExportAction">
   <property name="text">
    <string>Export...</string>
   </property>
   <property name="toolTip">
    <string>Export</string>
   </property>
  </action>
  <action name="mImportAction">
   <property name="text">
    <string>Import new...</string>
   </property>
   <property name="toolTip">
    <string>Import</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>mListView</sender>
   <signal>customContextMenuRequested(QPoint)</signal>
   <receiver>AttachmentsList</receiver>
   <slot>contextualMenu(QPoint)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mImportAction</sender>
   <signal>triggered()</signal>
   <receiver>AttachmentsList</receiver>
   <slot>importFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mExportAction</sender>
   <signal>triggered()</signal>
   <receiver>AttachmentsList</receiver>
   <slot>exportFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mDeleteAction</sender>
   <signal>triggered()</signal>
   <receiver>AttachmentsList</receiver>
   <slot>deleteFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mRenameAction</sender>
   <signal>triggered()</signal>
   <receiver>AttachmentsList</receiver>
   <slot>renameFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>contextualMenu(QPoint)</slot>
  <slot>importFile()</slot>
  <slot>exportFile()</slot>
  <slot>deleteFile()</slot>
  <slot>renameFile()</slot>
 </slots>
</ui>
